presto-server-rpm/src/main/rpm/postremove

```diff
@@ -10,6 +10,10 @@ then
     # rpm -e wont remove it, because this directory may later contain files not
     # deployed by the rpm
     rm -rf /var/lib/presto
+    # The RPM instead of linking to /usr/lib/presto/ as a directory attribute
+    # now links to all the individual jar files in /usr/lib/presto/lib/ and
+    # /usr/lib/presto/plugin/. So remove the parent dir manually during uninstall
+    rm -rf /usr/lib/presto
```

Comment:
RPM default behaviour is not to remove directories if user tampered with them. We could verify if we have only symlinks here, but I am not sure if it is not an overkill.